core/src/main/java/edu/wpi/grip/core/operations/python/PythonScriptFile.java

```diff
@@ -26,6 +28,32 @@
 @AutoValue
 public abstract class PythonScriptFile {
 
+  private static final Logger logger = Logger.getLogger(PythonScriptFile.class.getName());
+
+  /**
+   * Template for custom python operations. Includes imports for sockets, as well as OpenCV
+   * core and image processing classes.
+   *
+   * <p>The sample operation is a simple arithmetic "add" that hopefully shows how the script
+   * should be written.</p>
+   */
+  public static final String TEMPLATE =
+      "import edu.wpi.grip.core.sockets.SocketHints.Inputs as inputs\n"
+          + "import edu.wpi.grip.core.sockets.SocketHints.Outputs as outputs\n"
+          + "import org.bytedeco.javacpp.opencv_core as opencv_core\n"
+          + "import org.bytedeco.javacpp.opencv_imgproc as opencv_imgproc\n\n"
+          + "name = \"Addition Sample\"\n"
+          + "summary = \"The sample python operation to add two numbers\"\n\n"
+          + "inputs = [\n"
+          + "    inputs.createNumberSpinnerSocketHint(\"a\", 0.0),\n"
+          + "    inputs.createNumberSpinnerSocketHint(\"b\", 0.0),\n"
+          + "]\n"
+          + "outputs = [\n"
+          + "    outputs.createNumberSocketHint(\"sum\", 0.0),\n"
+          + "]\n\n\n" // two blank lines
+          + "def perform(a, b):\n"
+          + "    return a + b\n";
```

Comment:
Put it in a read once file.
Cache the string in memory after its loaded.
It will make it easier to edit in the future.